src/html.ejs

```diff
@@ -36,6 +36,8 @@
     <%- htmlWebpackPlugin.tags.headTags.map((originalTag) => { const tag = { ...originalTag, attributes: { ...originalTag.attributes } }; tag.attributes['src'] = "<" + "%= cdnUrl %" + ">" + originalTag.attributes['src']; return tag; }) %>
   <%% } %>
 
+  <%%- content.linkPreloadTags.join("") %>
```

Comment:
Since https://github.com/artsy/force/pull/14793 didn't allow for us to insert our CDN url, we ended up having to do this manually, which is fine!

Reply:
When I try this locally, I see a slew of new `<link rel="preload"` entries outputted here, similar to but different from the `<script defer` tags outputted just above. I'm not totally sure how this should work, but do we need both? They're basically opposites.

Locally I'm not sure if my CDN_URL is set up correctly but I see 2 separate requests for e.g. `runtime.*.js`, one of which is a CORS error.
<img width="922" alt="Screenshot 2024-11-05 at 2 38 26 PM" src="https://github.com/user-attachments/assets/35b398f2-7422-4dd9-aec8-959735090594">

Reply:
Script and link tags are different things. The link tags are hints to preload.

You can't use CDN_URL's locally during dev, it will error out. It'll work on staging.